.prettierignore

```diff
@@ -11,6 +11,7 @@
 !packages/**/
 !Specs/**/
 !Tools/**/
+!itwin-oauth-demo/**/
```

Comment:
To confirm my understanding: The plan is to remove these files before merging, correct? It's just for demo purposes until the ion route is in place?

Reply:
CC https://github.com/CesiumGS/cesium/pull/12289#discussion_r1850829211